lib/plugins/inspec-plugin-manager-cli/lib/inspec-plugin-manager-cli/cli_command.rb

```diff
@@ -74,15 +74,15 @@ def search(search_term) # rubocop: disable Metrics/AbcSize
         end
 
         puts
-        ui.bold(format(" %-30s%-50s", "Plugin Name", "Versions Available"))
+        ui.bold(format(" %-30s%-50s\n", "Plugin Name", "Versions Available"))
```

Comment:
Isn't that what `ui.line` does?

Reply:
No, `ui.line` makes a horizontal line. `ui.plain_line` does a `puts`, basically.